go.mod

```diff
@@ -1,24 +1,24 @@
 module github.com/aws/amazon-vpc-resource-controller-k8s
 
-go 1.13
+go 1.16
 
 require (
-	github.com/aws/amazon-vpc-cni-k8s v1.7.3
-	github.com/aws/aws-sdk-go v1.33.14
-	github.com/go-logr/logr v0.1.0
-	github.com/go-logr/zapr v0.1.0
+	github.com/aws/amazon-vpc-cni-k8s v1.9.0
+	github.com/aws/aws-sdk-go v1.40.6
```

Comment:
Do we need to switch this, given we are replacing this with internal?

Reply:
Will be updated after merging the PR for removing internal/aws-sdk-go.